apis/core-api/src/api/routes/v1/fanClubs/fanClubs/index.ts

```diff
@@ -0,0 +1,21 @@
+import Router from 'koa-router';
+import checkUserAuth from 'api/middlewares/check-user-auth';
+import checkFanClubExistence from 'api/middlewares/check-fan-club-existence';
+import ControllerAdapter from './ControllerAdapter';
+
+const idKey = 'fanClubId';
```

Comment:
or what is this, why this need

Reply:
did you mean `idKey` ? idKey is used because we have URLs like this:
`/fan-clubs/:fanClubId/memberships/:membershipId`

I need to specify to middleware which key is needed to check.